bundle/manifests/logging.openshift.io_elasticsearches_crd.yaml

```diff
@@ -0,0 +1,370 @@
+kind: CustomResourceDefinition
+metadata:
+  name: elasticsearches.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    kind: Elasticsearch
+    listKind: ElasticsearchList
+    plural: elasticsearches
+    singular: elasticsearch
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      description: Elasticsearch is the Schema for the elasticsearches API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: Specification of the desired behavior of the Elasticsearch
+            cluster
+          properties:
+            indexManagement:
+              description: Management spec for indicies
+              nullable: true
+              properties:
+                mappings:
+                  description: Mappings of policies to indicies
+                  items:
+                    description: IndexManagementPolicyMappingSpec maps a management
+                      policy to an index
+                    properties:
+                      aliases:
+                        description: Aliases to apply to a template
+                        items:
+                          type: string
+                        type: array
+                      name:
+                        description: The unique name of the policy mapping
+                        type: string
+                      policyRef:
+                        description: A reference to a defined policy
+                        type: string
+                    type: object
+                  type: array
+                policies:
+                  description: A list of polices for managing an indices
+                  items:
+                    description: IndexManagementPolicySpec is a definition of an index
+                      management policy
+                    properties:
+                      name:
+                        description: The unique name of the policy
+                        type: string
+                      phases:
+                        properties:
+                          delete:
+                            nullable: true
+                            properties:
+                              minAge:
```

Comment:
i realize we don't currently have these.. but would it be useful to add regex validation to this and the maxAge fields?

Reply:
Indeed we do server-side validation per regex on TimeUnit. This would be a good addition for client-side validation.